nixos/modules/services/continuous-integration/buildbot-master/default.nix

```diff
@@ -0,0 +1,243 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.buildbot-master;
+  configFile = pkgs.writeText "master.cfg" ''
+
+from buildbot.plugins import *
```

Comment:
There's no need to not indent this, by the way. The double quote thing looks for the smallest amount of leading whitespace and subtracts that from all lines, IIRC.